.travis.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     install:
       - pip install -r dev-requirements.txt
       - pip install -r requirements.txt
+      - pip install .
```

Comment:
What does this mean exactly? Just out of curiosity.

Reply:
This installs the honeybee-schema from setup.py which then will be used by `_get_honeybee_schema_version()` function. Otherwise it will fail since `honeybee-schema` is not installed before running the tests.